browser.go

```diff
@@ -270,6 +272,7 @@ func (b *Browser) PageFromTarget(targetID proto.TargetTargetID) (*Page, error) {
 	}
 
 	page = &Page{
+		e:         b.e,
```

Comment:
Inherit parent's handler

Reply:
Do you mean that this should be inherited into Element as well?

Reply:
Yes, exactly! Recursive design.

Reply:
This might help: https://github.com/go-rod/rod/blob/master/context.go